src/libraries/Microsoft.PowerFx.Core/Public/CheckResult.cs

```diff
@@ -289,7 +289,7 @@ public IEnumerable<ExpressionError> GetErrorsInLocale(CultureInfo culture)
         {
             culture ??= _defaultErrorCulture ?? ParserCultureInfo;
 
-            foreach (var error in this._errors)
+            foreach (var error in this._errors.Distinct(new ExpressionErrorComparer()))

```

Comment:
>this._errors.Distinct(new ExpressionErrorComparer())) [](http://example.com/codeflow?start=34&length=53)

What is the reason we get duplicate error? Is that fixable?

Reply:
One from the parser and one from the binder. This is the most straightforward and less invasive fix for the issue.

Reply:
Should we go to the binder if the parser fail?